jobs/build.Jenkinsfile

```
@@ -230,9 +230,38 @@ lock(resource: "build-${params.STREAM}") {
 
         // fetch from repos for the current build
         stage('Fetch') {
+            def overrides_fetch_param = ""
+            # Dont run this for production builds
+            recent_commits_to_lockfiles() {
+                // Checking if last commits meet the criteria
+                shwrap("""
+                # Last author time in UTC
+                echo "Last commit: $(date -ud @$(git log -1 --follow manifest-lock* --format="%ct"))" 
+                # Number of minutes to check
+                n=120
+                # Current time
+                current_time=$(date +%s)
+                # Time n minutes ago
+                n_minutes_ago=$(( current_time - ( 60 * n ) ))
+                # Last commit
+                last_git_log=$(git log -1 --follow manifest-lock* --format="%cI")
```

it appears we need  to use: 

```suggestion
                last_git_log=$(git log -1 --format="%cI" manifest-lock*)
```

The other ways gave errors:

```
[dustymabe@media fedora-coreos-config (upstreamtesting-devel %=)]$ git log -1 --follow manifest-lock* --format="%cI"
fatal: option '--format=%cI' must come before non-option arguments
[dustymabe@media fedora-coreos-config (upstreamtesting-devel %=)]$ git log -1 --follow --format="%cI" manifest-lock*
fatal: --follow requires exactly one pathspec
[dustymabe@media fedora-coreos-config (upstreamtesting-devel %=)]$ git log -1 --format="%cI" manifest-lock*
2023-05-05T03:03:10+00:00
```

---

also the output of `git log` isn't really giving us the time in the format we need it to be in to make the comparision you are making in the if statement below.

ok yes we need to use something like:

```
$ git log -1 --date=unix --format="%cd" manifest-lock*
1683255790
```

---

This is now amended.